specs/phase0/fork-choice.md

```diff
@@ -182,12 +182,13 @@ def get_latest_attesting_balance(store: Store, root: Root) -> Gwei:
             and get_ancestor(store, store.latest_messages[i].root, store.blocks[root].slot) == root)
     ))
     proposer_score = Gwei(0)
-    if store.proposer_boost_root != Root() and root == store.proposer_boost_root:
-        num_validators = len(get_active_validator_indices(state, get_current_epoch(state)))
-        avg_balance = get_total_active_balance(state) // num_validators
-        committee_size = num_validators // SLOTS_PER_EPOCH
-        committee_weight = committee_size * avg_balance
-        proposer_score = (committee_weight * PROPOSER_SCORE_BOOST) // 100
+    if store.proposer_boost_root != Root():
+        if get_ancestor(store, store.proposer_boost_root, store.blocks[root].slot) == root:
+            num_validators = len(get_active_validator_indices(state, get_current_epoch(state)))
+            avg_balance = get_total_active_balance(state) // num_validators
+            committee_size = num_validators // SLOTS_PER_EPOCH
+            committee_weight = committee_size * avg_balance
+            proposer_score = (committee_weight * PROPOSER_SCORE_BOOST) // 100
```

Comment:
It seems good to combine the two conditions so that we can reduce an indent.

```suggestion
    if (
        store.proposer_boost_root != Root()
        and get_ancestor(store, store.proposer_boost_root, store.blocks[root].slot) == root
    ):
        num_validators = len(get_active_validator_indices(state, get_current_epoch(state)))
        avg_balance = get_total_active_balance(state) // num_validators
        committee_size = num_validators // SLOTS_PER_EPOCH
        committee_weight = committee_size * avg_balance
        proposer_score = (committee_weight * PROPOSER_SCORE_BOOST) // 100
```


or more refactoring, but less efficient due to potential more `get_ancestor` calls:

```suggestion
    is_proposer_boost_root_ancestor = get_ancestor(store, store.proposer_boost_root, store.blocks[root].slot) == root
    if store.proposer_boost_root != Root() and is_proposer_boost_root_ancestor:
        num_validators = len(get_active_validator_indices(state, get_current_epoch(state)))
        avg_balance = get_total_active_balance(state) // num_validators
        committee_size = num_validators // SLOTS_PER_EPOCH
        committee_weight = committee_size * avg_balance
        proposer_score = (committee_weight * PROPOSER_SCORE_BOOST) // 100
```

Either way, since we were confused, could you add some comments about this condition? 🤣

Reply:
Yep, added [clarifying comment](https://github.com/ethereum/consensus-specs/pull/2760#discussion_r760747416) below.